app/src/main/java/org/mozilla/fenix/ext/Context.kt

```diff
@@ -89,3 +92,21 @@ fun Context.getStringWithArgSafe(@StringRes resId: Int, formatArg: String): Stri
  */
 val Context.accessibilityManager: AccessibilityManager get() =
     getSystemService(Context.ACCESSIBILITY_SERVICE) as AccessibilityManager
+
+/**
+ * Used to navigate to system notifications settings for app
+ */
+fun Context.navigateToNotificationsSettings() {
```

Comment:
Context.kt is getting quite bloated, I'd rather see a helper function to build the intent and leave the `startActivity` call inside the fragment.